packages/ndla-ui/src/Article/Article.tsx

```diff
@@ -250,23 +123,24 @@ export const Article = ({
     copyright?.creators.length || copyright?.rightsholders.length ? copyright.creators : copyright?.processors;
 
   return (
-    <ArticleWrapper modifier={modifier} data-ndla-article="">
-      <LayoutItem layout="center">
-        {messages.messageBox && (
-          <MSGboxWrapper>
-            <MessageBox links={messageBoxLinks}>{messages.messageBox}</MessageBox>
-          </MSGboxWrapper>
-        )}
-        <ArticleHeaderWrapper competenceGoals={competenceGoals}>
-          {heartButton ? <ArticleFavoritesButtonWrapper>{heartButton}</ArticleFavoritesButtonWrapper> : null}
-          <ArticleTitle id={id} icon={icon} label={messages.label} lang={lang}>
+    <ArticleWrapper data-ndla-article="">
+      <ArticleHeader>
+        <ArticleTitleWrapper>
+          {!!contentType && <ContentTypeBadgeNew contentType={contentType} />}
+          {!!heartButton && <ArticleFavoritesButtonWrapper>{heartButton}</ArticleFavoritesButtonWrapper>}
+          <Heading textStyle="heading.large" id={id} tabIndex={-1} lang={lang}>
             {title}
-          </ArticleTitle>
-          <ArticleIntroduction lang={lang}>{introduction}</ArticleIntroduction>
-        </ArticleHeaderWrapper>
-      </LayoutItem>
-      <LayoutItem layout="center">{content}</LayoutItem>
-      <LayoutItem layout="center">
+          </Heading>
+        </ArticleTitleWrapper>
+        {!!introduction && (
+          <Text lang={lang} textStyle="body.xlarge" asChild consumeCss>
+            <div>{introduction}</div>
```

Comment:
Hvorfor div her?

Reply:
`introduction` er HTML, og kan være mye rart. For eksempel `p`, eller to `p`-er. Dette må til for å unngå å neste paragrafer.